src/test/java/com/onfido/webhooks/WebhookClassicEventVerifierTest.java

```diff
@@ -27,7 +29,7 @@ public class WebhookEventVerifierTest extends TestBase {
                   .resourceType("check")
                   ._object(
                       new WebhookEventPayloadObject()
-                          .id(UUID.fromString("f2302f45-227d-413d-ad61-09ec077a086a"))
+                          .id(String.valueOf("f2302f45-227d-413d-ad61-09ec077a086a"))
```

Comment:
Is this needed? Can't we just just do `.id("f2302f45-227d-413d-ad61-09ec077a086a")`?